pyads/ads.py

```diff
@@ -319,6 +325,13 @@ def dict_from_bytes(
                         .decode("utf-8")
                     )
                     index += str_len + 1
+                elif plc_datatype == PLCTYPE_WSTRING:
+                    if str_len is None:
+                        str_len = 2 * PLC_DEFAULT_STRING_SIZE + 1  # use double default string size for WSTRING plus 1
+                    a = bytearray(byte_list[index: (index + (str_len + 2))])
```

Comment:
similar comment to above, we have a mix of +1 and +2 here, line 330 vs line 331

Reply:
Fixed it in f9f7ffe